files/build/scripts/git

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
```

Comment:
Please add file header comment to explain the purpose and usage.

Reply:
add header comment